docker/dev/docker-compose.yml

```diff
@@ -54,7 +54,7 @@ services:
       RATE_API_KEY: ${RATE_API_KEY}
       BASE_ASSET_SCALE: 2
       MAX_ASSET_SCALE: 9
-      WM_THRESHOLD: 100000000
+      RAPYD_THRESHOLD: 10000000 # 0.01
```

Comment:
I modified this to `0.01` (which is the lowest amount non-zero representable in asset scale 2) instead of `0.1`. Any reason to change it back?